tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -3201,6 +3223,36 @@ HRESULT CorProfiler::GenerateVoidILStartupMethod(const ModuleID module_id, mdMet
         return hr;
     }
 
+    /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    // Get a TypeRef for System.Char
+    mdTypeRef char_type_ref;
+    hr = metadata_emit->DefineTypeRefByName(corlib_ref, WStr("System.Char"), &char_type_ref);
+    if (FAILED(hr))
+    {
+        Logger::Warn("GenerateVoidILStartupMethod: DefineTypeRefByName::System.Char failed");
```

Comment:
Shouldn't all these Warns be Errs? If this fails something nasty has happened

Reply:
True, but the existing logs are all warnings. I can we can change that in another PR